web/src/auth/LoginPage.js

```diff
@@ -453,6 +478,33 @@ class LoginPage extends React.Component {
     }
   }
 
+  renderCaptchaModal(application) {
+    if (this.state.application && application?.providers?.length > 0) {
+      const provider = application.providers.filter(provider => {
+        return provider.provider?.category === "Captcha" && provider.provider?.type === "Default";
+      })[0];
+      // eslint-disable-next-line no-console
+      if (provider && provider.provider && provider?.rule === "Always") {
```

Comment:
provider can be removed?

Reply:
// eslint-disable-next-line no-console 
not provider